lib/rubocop/cop/root_cops/avoid_ruby_prof.rb

```diff
@@ -5,7 +5,7 @@ module Cop
     module RootCops
       class AvoidRubyProf < Cop
         ERROR = ":ruby_prof is for local use only and should not be committed."
-        FILE_NAME_MATCHER = /_spec\.rb\z/.freeze
```

Comment:
Are regexs frozen by default now? Or is `.freeze` no longer supported on them?

Reply:
They're frozen by default now